RELEASE-NOTES.md

```diff
@@ -1,8 +1,64 @@
 # Prefect Release Notes
 
+## Release 2.14.3
+
+### Observability with deployment status
+
+You can now track the status of your deployments in the Prefect UI, which is especially useful when serving flows as they have no associated work pool or worker. If you see a flow run enter a `LATE` state (it isn’t running), you can click into the deployment for that flow run and see a red indicator next to your deployment. The worker, runner, or agent polling that deployment or its associated work queue is offline.
+- Deployments created from served flows will have a `READY` status if its associated process is running.
+- Deployments created in a work pool will have a `READY` status when a worker is `ONLINE` and polling the associated work queue.
+- Deployments created in a push work pool (Prefect Cloud) will always have a `READY` status.
+  
+<p align="center">
+<img width="976" alt="a late flow run for a deployment that is `NOT_READY`" src="https://github.com/PrefectHQ/prefect/assets/42048900/db20979a-870d-44c4-ac0b-66f70d99e58b">
+</p>
+
+In Prefect Cloud, an event is emitted each time a deployment changes status. These events are viewable in the Event Feed.
+<p align="center">
+<img width="538" alt="event feed deployment status events" src="https://github.com/PrefectHQ/prefect/assets/42048900/8ee076cd-fd30-47d1-9ee5-6b5a3b383b63">
+</p>
+
+You can also create an automation triggered by deployment status changes on the Automations page!
+
+<p align="center">
+<img width="862" alt="deployment status trigger on automations page" src="https://github.com/PrefectHQ/prefect/assets/42048900/87a0945e-9b9e-406b-b020-fbd9733cb4c3">
+</p>
+
+### Enhancements
+- Add option to skip building a Docker image with `flow.deploy` — https://github.com/PrefectHQ/prefect/pull/11082
+- Add ability to specify remote storage for `Flow.from_source` via ffspec - https://github.com/PrefectHQ/prefect/pull/11072
+- Add variables page empty state — https://github.com/PrefectHQ/prefect/pull/11044
+- Allow composite sort of block documents by `block_type_name` and name — https://github.com/PrefectHQ/prefect/pull/11054
+- Warn if task parameter introspection takes a long time — https://github.com/PrefectHQ/prefect/pull/11075
+
+### Fixes
+- Update cancellation cleanup service to allow for infrastructure teardown — https://github.com/PrefectHQ/prefect/pull/11055
+- Allow `password` to be provided in `credentials` for `GitRespository` — https://github.com/PrefectHQ/prefect/pull/11056
+- Reset `PREFECT_UI_SERVE_BASE` to enable page refresh loading for non dashboard pages — https://github.com/PrefectHQ/prefect/pull/11065
+- Skips unsupported Windows tests — https://github.com/PrefectHQ/prefect/pull/11076
+- Allow runner to load remotely stored flows when running hooks — https://github.com/PrefectHQ/prefect/pull/11077
+- Avoid yanked `pytest-asyncio==0.22.0` — https://github.com/PrefectHQ/prefect/pull/11064
```

Comment:
I think this can be at the bottom of fixes as it's technically maintenance but we don't use that category

Reply:
yeah good call